src/Protocol/Protocol.h

```diff
@@ -15,6 +15,7 @@
 #include "../Map.h"
 #include "../ByteBuffer.h"
 #include "../EffectID.h"
+#include "../ClientHandle.h"
```

Comment:
Is this include still needed? There's no other change in Protocol.h, so I'd assume it's not.